packages/plugin-react/CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 ## Unreleased
 
+### Add backwards compatible option `reactRefreshPrefix`
+
+Add backwards compatible option `reactRefreshPrefix` to set React refresh runtime url prefix.
+This is useful in module federation context to enable HMR by setting the host url on a vite config which is serving a remote app.
+See full discussion here: https://github.com/module-federation/vite/issues/183#issuecomment-2751825367
+
+```ts
+export default defineConfig(({ command }) => {
+  return {
+    plugins: [react({ reactRefreshPrefix: 'http://localhost:3000' })],
+  }
+})
```

Comment:
This example doesn't require the function callback, better to keep it in the simplest form
```suggestion
export default defineConfig({
  plugins: [react({ reactRefreshPrefix: 'http://localhost:3000' })],
})
```